### PR TITLE
chore: update qg-checklist

### PR DESCRIPTION
**Pull request by FaGru3n**
## Description
- remove colapsible items and remove all sub-tasks
- add link to our automatic checks
- add back on top and back on section links

updates #14 

## Visual preview
image

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Comment by SebastianBezold** [comment]
Missing update to 4.02 in my opinion: It's also allowed to (maybe even encouraged) to have a dedicated .md file for DockerHub. Title could therefore be misleading

**Comment by almadigabor** [comment]
This is the template I created temporarily: https://github.com/eclipse-tractusx/item-relationship-service/issues/277

**Comment by FaGru3n** [comment]
updated the Checklist on @almadigabor suggestions so that now sub-task is show on the template but for earch TRG a available link is present what to check in there... only thing we can still discuss is the wording.

File: .github/ISSUE_TEMPLATE/system-team-qg-checklist.md [hunk 1]
```diff
@@ -22,159 +22,73 @@ QG5 Approval: yes/no <!-- Note: Please ask for the approval from Release Managem
 
 ### Check of Tractus-X Release Guidelines
 
-This QG x Check is depending on the mandatory information from our current [Release Guidelines](https://eclipse-tractusx.github.io/docs/release).
+- Currently implemented automatic checks can be found under your product on our [Release Guidelines Checks Board](https://eclipse-tractusx.github.io/sig-release/) 
+- This QG x Check is depending on the mandatory information from our current [Release Guidelines](https://eclipse-tractusx.github.io/docs/release)
```
Comment (SebastianBezold) [inline .github/ISSUE_TEMPLATE/system-team-qg-checklist.md]: would remove the 'x' after QG

File: .github/ISSUE_TEMPLATE/system-team-qg-checklist.md [hunk 2]
```diff
@@ -22,159 +22,73 @@ QG5 Approval: yes/no <!-- Note: Please ask for the approval from Release Managem
...
-  </details>
-- [ ] [TRG 5.11](https://eclipse-tractusx.github.io/docs/release/trg-5/trg-5-11) Upgradeability PRERELEASE
-  <details>
-  <summary>Checks within TRG 5.11</summary>
-
-  - [ ] TRG 5.11 Based on the Helm test workflow, you must provide a GitHub action which takes the latest released helm chart, does an installation of it and then execute the upgrade to the current / new version.
-  </details>
-  
-#### TRG 6 Released Helm Chart 
+- [ ] [TRG 5.11](https://eclipse-tractusx.github.io/docs/release/trg-5/trg-5-11) Upgrade-ability PRERELEASE
```
Comment (SebastianBezold) [inline .github/ISSUE_TEMPLATE/system-team-qg-checklist.md]: Upgradeability as one word. Is it still 'PRERELEASE'?

**Review by SebastianBezold — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/system-team-qg-checklist.md (hunk 2 above)
Comment (FaGru3n) [reply .github/ISSUE_TEMPLATE/system-team-qg-checklist.md]: Hi @SebastianBezold don´t know, do you have more info´s if it is still fit our agenda?

File: .github/ISSUE_TEMPLATE/system-team-qg-checklist.md (hunk 1 above)
Comment (FaGru3n) [reply .github/ISSUE_TEMPLATE/system-team-qg-checklist.md]: [chore: fix typo + reduce x + removed PRERELEASE](https://github.com/eclipse-tractusx/.github/pull/15/commits/5577a78c452beadae1d5b7887dd6fc0931164617)

File: .github/ISSUE_TEMPLATE/system-team-qg-checklist.md (hunk 2 above)
Comment (FaGru3n) [reply .github/ISSUE_TEMPLATE/system-team-qg-checklist.md]: [chore: fix typo + reduce x + removed PRERELEASE](https://github.com/eclipse-tractusx/.github/pull/15/commits/5577a78c452beadae1d5b7887dd6fc0931164617)

removed it

Comment (FaGru3n) [reply .github/ISSUE_TEMPLATE/system-team-qg-checklist.md]: current TRG tells us "you must provide a GitHub action which takes the latest released helm chartuse the Helm test workflow"

**Review by SebastianBezold — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/system-team-qg-checklist.md [hunk 3]
```diff
@@ -22,159 +22,73 @@ QG5 Approval: yes/no <!-- Note: Please ask for the approval from Release Managem
...
 ### Check of Tractus-X Release Guidelines
 
-This QG x Check is depending on the mandatory information from our current [Release Guidelines](https://eclipse-tractusx.github.io/docs/release).
+- Currently implemented automatic checks can be found under your product on our [Release Guidelines Checks Board](https://eclipse-tractusx.github.io/sig-release/) 
+- This QG Check is depending on the mandatory information from our current [Release Guidelines](https://eclipse-tractusx.github.io/docs/release)
 
 #### TRG 1 Documentation
+
 - [ ] [TRG 1.01](https://eclipse-tractusx.github.io/docs/release/trg-1/trg-1-1) appropriate `README.md` 
 - [ ] [TRG 1.02](https://eclipse-tractusx.github.io/docs/release/trg-1/trg-1-2) appropriate `INSTALL.md`
```
Comment (tomaszbarwicki) [inline .github/ISSUE_TEMPLATE/system-team-qg-checklist.md]: Do we need a line for optional check?

File: .github/ISSUE_TEMPLATE/system-team-qg-checklist.md [hunk 4]
```diff
@@ -22,159 +22,73 @@ QG5 Approval: yes/no <!-- Note: Please ask for the approval from Release Managem
...
-  - [ ] TRG 2.03 `AUTHORS.md` file (optional) (TRG 2.03)
-  - [ ] TRG 2.03 `CODE_OF_CONDUCT.md` file (TRG 2.03)
-  - [ ] TRG 2.03 `CONTRIBUTING.md` file (TRG 2.03)
-  - [ ] TRG 2.03 `DEPENDENCIES` file(s) with up to date content (Dash tool generated) (TRG 2.03)
-  - [ ] TRG 2.03 `LICENSE` file (TRG 2.03)
-  - [ ] TRG 2.03 `NOTICE.md` file (TRG 2.03)
-  - [ ] TRG 2.03 `SECURITY.md` file (TRG 2.03)
-  </details>
-
 - [ ] [TRG 2.04](https://eclipse-tractusx.github.io/docs/release/trg-2/trg-2-4) Leading product repository
```
Comment (tomaszbarwicki) [inline .github/ISSUE_TEMPLATE/system-team-qg-checklist.md]: minor thing but some lines we start lower case, some uppercase, Leading or leading? would stick to one

File: .github/ISSUE_TEMPLATE/system-team-qg-checklist.md [hunk 5]
```diff
@@ -22,159 +22,73 @@ QG5 Approval: yes/no <!-- Note: Please ask for the approval from Release Managem
...
-- [ ] [TRG 4.02](https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-2) top level `README.md` file, that contains information about the used base image <!-- Java, Kotlin, ... if JVM based language use base image from [Eclipse Temurin](https://hub.docker.com/_/eclipse-temurin) -->
+- [ ] [TRG 4.02](https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-2) top level `README.md` file or separate `README.md` file for DockerHub, that contains information about the used base image   <!-- Java, Kotlin, ... if JVM based language use base image from [Eclipse Temurin](https://hub.docker.com/_/eclipse-temurin) -->
 - [ ] [TRG 4.03](https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-3) Image has `USER` command and Non Root Container
-  <details>
-  <summary>Checks within TRG 4.03</summary>
-  
-  - [ ] TRG 4.03 `deployment.yaml` has `runAsUser` and `allowPrivilegeEscalation: false` properly set
-  </details>
-
 - [ ] [TRG 4.05](https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-05) released image must be place `DockerHub` as mandatory container registry; remove `GHCR` references
```
Comment (tomaszbarwicki) [inline .github/ISSUE_TEMPLATE/system-team-qg-checklist.md]: perhaps 'released image must be placed at Dockerhub' might be more correct here, would also replace semicolon with comma as in other lines

File: .github/ISSUE_TEMPLATE/system-team-qg-checklist.md [hunk 6]
```diff
@@ -22,159 +22,73 @@ QG5 Approval: yes/no <!-- Note: Please ask for the approval from Release Managem
...
-        Chart.yaml
-        ...
-  AUTHORS.md 
-  DEPENDENCIES.md 
-  LICENCE 
-  README.md 
-  ```
-  </details>
-
 - [ ] [TRG 5.04](https://eclipse-tractusx.github.io/docs/release/trg-5/trg-5-04) CPU and memory limits and requests are properly set
```
Comment (tomaszbarwicki) [inline .github/ISSUE_TEMPLATE/system-team-qg-checklist.md]: maybe 'limits and requests set for CPU & Mem' ?

File: .github/ISSUE_TEMPLATE/system-team-qg-checklist.md [hunk 7]
```diff
@@ -22,159 +22,73 @@ QG5 Approval: yes/no <!-- Note: Please ask for the approval from Release Managem
...
 - [ ] [TRG 5.06](https://eclipse-tractusx.github.io/docs/release/trg-5/trg-5-06) application must be configurable through the Helm chart <!-- every startup configuration aspect of your application must be configurable through the Helm chart (ingress class, tls, labels, annotations, database, secrets, persistence, env variables) -->
 - [ ] [TRG 5.07](https://eclipse-tractusx.github.io/docs/release/trg-5/trg-5-07) dependencies are present in the `Chart.yaml` they are properly configured
```
Comment (tomaszbarwicki) [inline .github/ISSUE_TEMPLATE/system-team-qg-checklist.md]: suggest "dependencies are present and properly configured in the `Chart.yaml`"

**Review by tomaszbarwicki — APPROVED** [review]
In general template looks good to me, just some suggestions here and there.

File: .github/ISSUE_TEMPLATE/system-team-qg-checklist.md (hunk 3 above)
Comment (FaGru3n) [reply .github/ISSUE_TEMPLATE/system-team-qg-checklist.md]: do you mean install is optional? if it proper declared in `README.md`?

File: .github/ISSUE_TEMPLATE/system-team-qg-checklist.md (hunk 7 above)
Comment (FaGru3n) [reply .github/ISSUE_TEMPLATE/system-team-qg-checklist.md]: [b21738e](https://github.com/eclipse-tractusx/.github/pull/15/commits/b21738e1598f04178b888648a668ad427094ffbc)

File: .github/ISSUE_TEMPLATE/system-team-qg-checklist.md (hunk 6 above)
Comment (FaGru3n) [reply .github/ISSUE_TEMPLATE/system-team-qg-checklist.md]: [b21738e](https://github.com/eclipse-tractusx/.github/pull/15/commits/b21738e1598f04178b888648a668ad427094ffbc)

File: .github/ISSUE_TEMPLATE/system-team-qg-checklist.md (hunk 5 above)
Comment (FaGru3n) [reply .github/ISSUE_TEMPLATE/system-team-qg-checklist.md]: [b21738e](https://github.com/eclipse-tractusx/.github/pull/15/commits/b21738e1598f04178b888648a668ad427094ffbc)

File: .github/ISSUE_TEMPLATE/system-team-qg-checklist.md (hunk 4 above)
Comment (FaGru3n) [reply .github/ISSUE_TEMPLATE/system-team-qg-checklist.md]: [b21738e](https://github.com/eclipse-tractusx/.github/pull/15/commits/b21738e1598f04178b888648a668ad427094ffbc)

File: .github/ISSUE_TEMPLATE/system-team-qg-checklist.md (hunk 3 above)
Comment (FaGru3n) [reply .github/ISSUE_TEMPLATE/system-team-qg-checklist.md]: [23a6ed9](https://github.com/eclipse-tractusx/.github/pull/15/commits/23a6ed930f45b4ce3ff3800d01f8937a695c52eb) but open for discussion

**Pull request merged**